tests/basic/pure_test.go

```diff
@@ -1014,3 +1014,127 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+// This test Kills the PX nodes where FADA volumes are attached, Deletes the pods and PVCs.
+/*
+https://portworx.testrail.net/index.php?/tests/view/72760884
+
+*/
+var _ = Describe("{AppCleanUpWhenPxKill}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("AppCleanUpWhenPxKill", "Test creates multiple FADA volume and kills px nodes while the pods and pvc's are being deleted", nil, 72760884)
+	})
+	It("Schedules apps that use FADA volumes, kill the nodes where these volumes are placed while the volumes are being deleted.", func() {
+		var contexts = make([]*scheduler.Context, 0)
+		var wg sync.WaitGroup
+		//Scheduling app with volume placement strategy
+		applist := Inst().AppList
+		rand.Seed(time.Now().Unix())
+
+		//select the one storage node,one storageless node and one KVDB member node to place volumes and kill the nodes while apps are being destroyed
+		storageNodes := node.GetStorageNodes()
+		storageLessNodes := node.GetStorageLessNodes()
+		kvdbNodes, err := GetAllKvdbNodes()
+		log.FailOnError(err, "Failed to get kvdb nodes")
+		var selectedNodes []node.Node
+		selectedNodes = append(selectedNodes, storageNodes[rand.Intn(len(storageNodes))])
+		selectedNodes = append(selectedNodes, storageLessNodes[rand.Intn(len(storageLessNodes))])
+		selectedKvdbNode, err := node.GetNodeDetailsByNodeID(kvdbNodes[rand.Intn(len(kvdbNodes))].ID)
+		log.FailOnError(err, "Failed to get kvdb node details")
+		for _, kvdbNode := range kvdbNodes {
+			if kvdbNode.ID != selectedNodes[0].Id && kvdbNode.ID != selectedNodes[1].Id {
```

Comment:
just add one check with selectedNodes[0].Id and add kvdb node. you can remove line 1042

Reply:
Done